📝 8.5.0 bug fixes (#329)
Instabug/Instabug-React-Native: CHANGELOG.md

## v8.5.1 (2019-07-22)

> * Fixes an issue when causing duplicate framework when using cocoapods

Reviewer: ```suggestion
* Fixes an issue that would cause Instabug.framework to appear twice when using CocoaPods.
```

> * Fixes an issue that cause a deadlock when calling `concole.log` many times after `Instabug.startWithToken`

Reviewer: ```suggestion
* Fixes a deadlock that would happen when `console.log` is called immediately after `startWithToken`.
```

> * Fixes an issue in the upload sourcemap script that was preventing it from greping the token automatically

Reviewer: ```suggestion
* Fixes an issue that prevented app token from being detected correctly when uploading source map files.
```
